### PR TITLE
UI Accessibility Updates

### PR DESCRIPTION
**Pull request by ElDuke717**
Made changes to the UI to improve accessibility.

### CODE REVIEW

File: .babelrc
Comment (j-mccoll) [inline .babelrc:1]: Remove this change

File: test_output1
Comment (j-mccoll) [inline test_output1:1]: Delete this test output file

File: .babelrc
Comment (j-mccoll) [inline .babelrc:1]: Remove this change

File: src/app/components/StateRoute/StateRoute.tsx
Comment (j-mccoll) [inline src/app/components/StateRoute/StateRoute.tsx:1]: Let's remove these console logs before merging into dev

File: src/backend/routers/snapShot.ts
Comment (j-mccoll) [inline src/backend/routers/snapShot.ts:1]: Remove these changes before committing

**Review by j-mccoll — CHANGES_REQUESTED** [review]
Styling looks great Nick! I noted several files where you'll want to make changes before merging this PR.

**Comment by ElDuke717** [comment]
Going to try another after updates.

File: src/backend/controllers/createTree.ts
[the remark this answers is not in the document]
Comment (j-mccoll) [reply src/backend/controllers/createTree.ts:1]: Sorry, I think I wasn't clear! Let's leave in the original comments (line 155-190). I was just saying let's remove the extra space in line 35, so there's nothing to change in this file.

**Review by j-mccoll — CHANGES_REQUESTED** [review]
If you could please update createTree.ts and snapShot.ts so they're identical to the current version in dev, then I think we're set!

**Pull request closed without merging**